public/app/features/transformers/helpers/rowsToFieldsHelper.ts

```diff
@@ -0,0 +1,62 @@
+export const rowsToFieldsHelper = () => {
+  return `
+  Use this transformation to convert rows into separate fields. This can be useful as fields can be styled and configured individually. It can also use additional fields as sources for dynamic field configuration or map them to field labels. The additional labels can then be used to define better display names for the resulting fields.
```

Comment:
```suggestion
  Use this transformation to convert rows into separate fields. This can be useful because fields can be styled and configured individually. It can also use additional fields as sources for dynamic field configuration or map them to field labels. The additional labels can then be used to define better display names for the resulting fields.
```